_episodes/05-history.md

```diff
@@ -409,6 +412,16 @@ moving backward and forward in time becomes much easier.
 > 4. Type in the new commit message.
 >
 > 5. Save and close
+> 
+> 
+> > ## Solution
+> > 
+> > The command `git log` lists project history with commit IDs.  
+> > 
+> > The command `git show HEAD~1` shows changes made at the commit one before `HEAD` and lists
```

Comment:
Maybe I'm misinterpreting the exercise prompt, but wouldn't her last commit to the project repository be shown with `git show HEAD`, not `git show HEAD~1`?

Reply:
Yes, I think when I wrote this I was thinking `checkout` instead of `revert`.  I fixed the error.